ui/perfherder/alerts/AlertsViewControls.jsx

```diff
@@ -127,15 +122,22 @@ AlertsViewControls.propTypes = {
   validated: PropTypes.shape({
     updateParams: PropTypes.func,
   }).isRequired,
-  isListingAlertSummaries: PropTypes.bool,
-  dropdownOptions: PropTypes.arrayOf(PropTypes.shape({})),
+  isListMode: PropTypes.bool.isRequired,
+  filterText: PropTypes.string.isRequired,
+  hideImprovements: PropTypes.bool.isRequired,
+  hideDownstream: PropTypes.bool.isRequired,
+  hideAssignedToOthers: PropTypes.bool.isRequired,
+  framework: PropTypes.shape({}).isRequired,
+  status: PropTypes.string.isRequired,
+  // TODO: rename to ~updateFilters
```

Comment:
This TODO is needed?

Reply:
still thinking of a better name for that function :)